admin/kernel-masterdata-service/docs/Configuration.md

```diff
@@ -0,0 +1,60 @@
+# Masterdata services Configuration Guide
+## Overview
+The guide here lists down some of the important properties that may be customised for a given installation. Note that the listing here is not exhaustive, but a checklist to review properties that are likely to be different from default.  If you would like to see all the properites, then refer to the files listed below.
+
+## Configuration files
+Masterdata services uses the following configuration files:
+```
+application-default.properties
+kernel-default.properties
+```
+The above files are located in [mosip-config](https://github.com/mosip/mosip-config/blob/develop2-v2/) repo
+
+## DB
+* `masterdata_database_username`
+* `masterdata_database_password`
+* `masterdata_database_url`
+
+Point the above to your DB .  Default is set to point to in-cluster Postgres installed with sandbox.
+
+## Admin UI config
+* `mosip.admin.ui.configs`
+Above property is used for UI configuration of admin. Value of the property is a key value pairs. Example:
+
+* `mosip.admin.ui.configsversion=version:${application.configuration.level.version};HierarchyLevel:${mosip.recommended.centers.locCode}`
+
+## Value limit on fields
+* `mosip.kernel.registrationcenterid.length:specify length of registration center id`
+* `mosip.kernel.machineid.length:specify length of machine id`
+* `mosip.recommended.centers.locCode:specify hierarchy location code`
+* `master.search.maximum.rows:specify maximum number of rows to be fetched in the search and filter queries`
+* `masterdata.registerdevice.timestamp.validate:specify value with + or - to validate device time stamp`
+* `application.configuration.level.version:specify version of application`
+* `mosip.kernel.applicant.type.age.limit:specify adult age limit this differs based on country`
```

Comment:
mosip.kernel.applicant.type.age.limit is this property still in use ?

Reply:
Earlier this  was used in mvel file ,cross verified ,it is not used in mvel file now logic is changed will remove this property